app/pb_kits/playbook/pb_typeahead/components/MultiValue.jsx

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+
+import React from 'react'
+import { components } from 'react-select'
+
+import { FormPill } from '../../'
+
+type Props = {
+  data: object,
+  multiValueTemplate: any,
+  removeProps: any,
+}
+
+const MultiValue = (props: Props) => {
+  const {
+    data,
+    removeProps,
+  } = props
+
+  const handleOnClick = removeProps.onClick
+
+  return (
+    <components.MultiValueContainer
+        className="text_input_multivalue_container"
+        {...props}
+    >
+      <If condition={data.avatarUrl}>
+        <FormPill
+            avatarUrl={data.avatarUrl}
+            marginRight="xs"
+            name={data.label}
+            onClick={handleOnClick}
```

Comment:
Can I add my own callback to this `onClick` event? 

An example of usage, in the Audience Selector, when we remove a user from the typeahead we make a request to the backend to update the DB without having to click in a submit button.

Reply:
I can certainly build that in at the `selectProps` level. Maybe we can name that `onMultiValueClick`? wdyt?

Reply:
I like that!

Reply:
@gmfvpereira done!